ironfish-rust-nodejs/src/structs/transaction.rs

```diff
@@ -327,7 +327,7 @@ impl NativeTransaction {
     #[napi]
     pub fn build(
         &mut self,
-        proof_generation_key_str: String,
+        proof_authorizing_key: String,
```

Comment:
It might be tricky to change this interface before changing the account structure -- right now the multisig integration test depends on using the `proofGenerationKey` that's returned in the trusted dealer key package

Reply:
i was working through changing the account structure and kept running into performing hacks to get the authorizing key. I figured it might make more sense to fix this first because changing the account structure that has larger implications: https://github.com/iron-fish/ironfish/pull/4636

Reply:
Yeah changing the account structure is non-trivial. I meant more that we can make the change to the key object to return nsk instead of pgk without changing this interface